packages/siwe/src/requests.tsx

```diff
@@ -0,0 +1,74 @@
+export interface NonceResponse {
+  nonce: string
+  ok: boolean
+}
+
+export interface AuthResponse {
+  address: string
+  ok: boolean
+}
+
+export interface LogoutResponse {
+  ok: boolean
+}
+
+export interface SiweFetchers {
+  getNonce: () => Promise<NonceResponse>
+  getAuth: () => Promise<AuthResponse>
+  login: (signature: string, message: string) => Promise<AuthResponse>
+  logout: () => Promise<LogoutResponse>
+}
+
+export const getFetchers = (backendUrl: string): SiweFetchers => {
+  return {
+    getAuth: async () => {
+      const authRequest = await fetch(`${backendUrl}/auth/me`, {
+        method: 'GET',
+        credentials: 'include',
+      })
+      const authResponse = await authRequest.json()
+    
+      return {
+        ...authResponse,
+        ...authRequest,
```

Comment:
why merge them?

Reply:
to have request.ok and full response data